### PR TITLE
assignments complete with pep8

### PR DESCRIPTION
**Pull request by fuse999**

### CODE REVIEW

File: Basic-python-demo.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import calendar
```
Comment (AdnanCodes) [inline Basic-python-demo.py:1]: Great work making your own Demo

File: src/02_datatypes.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
 # Write a print statement that combines x + y into the integer value 12
 
-# YOUR CODE HERE
+print(5 + int(y))
 
 
 # Write a print statement that combines x + y into the string value 57
```
Comment (AdnanCodes) [inline src/02_datatypes.py:19]: You missed a little bit of the problem.

File: src/05_lists.py [hunk 3]
```diff
@@ -7,23 +7,24 @@
...
 
 # Print the length of list x
-# YOUR CODE HERE
+print(len(x))
 
 # Print all the values in x multiplied by 1000
-# YOUR CODE HERE
\ No newline at end of file
+print((x[0]*1000), (x[1]*1000), (x[2]*1000),
+      (x[3]*1000), (x[4]*1000), (x[5]*1000), (x[6]*1000))
```
Comment (AdnanCodes) [inline src/05_lists.py:30]: That's one way to do it, can you do it programmatically?

File: src/06_tuples.py [hunk 4]
```diff
@@ -17,28 +17,33 @@
...
+    return print(result)
+
 
 t = (1, 2, 5, 7, 99)
 print_tuple(t)  # Prints 1 2 5 7 99, one per line
 
 # Declare a tuple of 1 element then print it
-u = (1)  # What needs to be added to make this work?
-print_tuple(u)
+u = [(1)]  # What needs to be added to make this work? []
```
Comment (AdnanCodes) [inline src/06_tuples.py:48]: Forcing a list to bypass your function is clever, still, remember how math expressions are evaluated in python

File: src/08_comprehensions.py [hunk 5]
```diff
@@ -27,15 +29,15 @@
...
 
 # Use a list comprehension to create a list containing only the _even_ elements
 # the user entered into list x.
 
 x = input("Enter comma-separated numbers: ").split(',')
-
 # What do you need between the square brackets to make it work?
-y = []
-
+y = [i for i in x if int(i) % 2 == 0]
```
Comment (AdnanCodes) [inline src/08_comprehensions.py:42]: You can refactor the other code into List comprehensions.

File: src/15_classes.py [hunk 6]
```diff
@@ -1,29 +1,43 @@
...
 
-# YOUR CODE HERE
-
 # Print it--also make this print more nicely
-print(geocache)
+# print(geocache)
+
+if __name__ == "__main__":
+    new_waypoint = WayPoint("Catacombs",41.70505,-121.51521)
+    print(f'"{new_waypoint.name}", {new_waypoint.lat}, {new_waypoint.lon}')
```
Comment (AdnanCodes) [inline src/15_classes.py:40]: See if you can use __str__ method

File: src/bar.txt [hunk 7]
```diff
@@ -0,0 +1,3 @@
+This is a test
+This is a second test
+This is a third test
```
Comment (AdnanCodes) [inline src/bar.txt:3]: May be it is cool test

File: Basic-python-demo.py (hunk 1 above)
Comment (fuse999) [reply Basic-python-demo.py:1]: great thankyou

Comment (fuse999) [reply Basic-python-demo.py:1]: @AdnanCodes  Hellp!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Pull request merged**